cherry_picker/cherry_picker.py

```diff
@@ -196,7 +196,7 @@ def username(self):
         cmd = ["git", "config", "--get", f"remote.{self.pr_remote}.url"]
         result = self.run_cmd(cmd, required_real_result=True)
         # implicit ssh URIs use : to separate host from user, others just use /
-        username = result.replace(":", "/").split("/")[-2]
+        username = result.replace(":", "/").removesuffix("/").split("/")[-2]
```

Comment:
I didn't realise ``cherry_picker`` still supported 3.8. Perhaps:

```suggestion
        username = result.replace(":", "/").rstrip("/").split("/")[-2]
```

Reply:
We could consider dropping 3.8, but that looks a good enough replacement.

Please could you also add a test case to `test_username`?